File: Bubble Sort/bubble_sort.py
arr = [11,1,45,33,2,13,15]

def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        swapped = True
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
        if swapped == False:
            break

    return arr

print(bubble_sort(arr))

File: Bubble Sort/BubbleSort.js
function BubbleSort(arr) {
    for (let i = arr.length; i > 0; i--) {
        let swapped = false;
        for (let j = 0; j < i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
                swapped = true;
            }
        }
        if (swapped == false) {
            break;
        }
    }
    return arr;
}

console.log(BubbleSort([18, 1, 43, 23, 19, 21, 11]));